src/main/java/de/tum/cit/aet/artemis/programming/service/hestia/TestwiseCoverageService.java

```diff
@@ -225,7 +225,7 @@ private Map<String, Integer> getLineCountByFilePath(ProgrammingSubmission submis
             var solutionRepo = gitService.getOrCheckoutRepository(solutionParticipation.getVcsRepositoryUri(), true);
             gitService.resetToOriginHead(solutionRepo);
             gitService.pullIgnoreConflicts(solutionRepo);
-            var solutionFiles = repositoryService.getFilesContentFromWorkingCopy(solutionRepo);
+            var solutionFiles = repositoryService.getFilesContentFromWorkingCopy(solutionRepo, false);
```

Comment:
maybe we could overload the method with a list of argument that only contains the `Repository`? So we can use `repositoryService.getFilesFromWorkingCopy(solutionRepo)`?